lms/services/canvas_api.py

```diff
@@ -33,202 +48,351 @@ def get_token(self, authorization_code):
         """
         Get an access token for the current LTI user.
 
-        Posts to the Canvas API to get the access token and returns it.
-
-        :arg authorization_code: The Canvas API OAuth 2.0 authorization code to
+        :param authorization_code: The Canvas API OAuth 2.0 authorization code to
             exchange for an access token
-        :type authorization_code: str
```

Comment:
We aren't mentioning parameter data types, because it doesn't do anything, and also this is Python